.gitignore

```diff
@@ -1,3 +1,44 @@
+# Created by https://www.gitignore.io/api/windows,laravel
+
+### Windows ###
+# Windows image file caches
+Thumbs.db
+ehthumbs.db
+
+# Folder config file
+Desktop.ini
+
+# Recycle Bin used on file shares
+$RECYCLE.BIN/
+
+# Windows Installer files
+*.cab
+*.msi
+*.msm
+*.msp
+
+# Windows shortcuts
+*.lnk
```

Comment:
Everything from the start of the file to here are window related. I don't use window. Can you provide details on what are they and why it should be here.